src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -44,36 +44,66 @@ const ActorInit En_Door_InitVars = {
     (ActorFunc)EnDoor_Draw,
 };
 
+typedef struct {
+    /* 0x00 */ s16 sceneNum;
+    /* 0x02 */ u8 dListIndex;
+    /* 0x04 */ s16 objectId;
+} EnDoorInfo;
+
+typedef enum {
+    /* 0 */ DOOR_DL_GAMEPLAY_KEEP,
+    /* 1 */ DOOR_DL_HIDAN,
+    /* 2 */ DOOR_DL_MIZU,
+    /* 3 */ DOOR_DL_HAKA,
```

Comment:
Do we need to stick with the Japanese names for this enum? I feel they could just be made English? I guess `HAKA` would be both shadow and bottom of the well right?

Reply:
Nothing came out of the discussion I initiated on renaming scenes to english, so I'm using the debug names by default :shrug:

Ideally yeah I would want to stop using the debug names

Reply:
I was thinking more of this specific enums instead of the scenes themselves. Actors for example use english names in their enums, even if the name of the actor is still Japanese/OG. Not a big deal though, it'll get done eventually

Reply:
Well I mean if we name the scenes something we may need to update the english names we arbitrarily went with everywhere. Maybe if we don't actually rename the scenes we should at least agree on their future names then?

Reply:
in this case I would go with english names, but can see how that inconsistent is an issue for some